README.md

```diff
@@ -52,11 +53,12 @@ bash build/build.sh
 
 # At this point, the HTML & PDF outputs will have been created. The remaining
 # commands are for serving the webpage to view the HTML manuscript locally.
+# This is required to view local images in the HTML output.
 
 # Configure the webpage directory
 python build/webpage.py
 
-# View the manuscript locally at http://localhost:8000/
+# View the manuscript webpage/index.html locally at http://localhost:8000/
```

Comment:
We should probably note that usually you should be able to just open `webpage/index.html` in a browser and it will work. However, to open a local webserver use the following:


This has changed from before. The HTML outputs now work better when just openning in browser without web server.